### DISCUSSION TITLE
what is your conclusion with rtklib PPP solution

### DISCUSSION THREAD

**Discussion by yydgis** [AdrianKriger/realTimePPP101#8] [General]
I checked your solutions, it seems that you cannot get cm positioning accuracy after a few hours convergence. Any comments about it?

**Comment by AdrianKriger** [comment]
Thank you for reaching out @yydgis. I am grateful.

Forgive my ignorance. Can you tell me which observation files and reference you were using please?
Are you referring to the [SRV370S folder](https://github.com/AdrianKriger/realTimePPP101/tree/main/SRV370S)?

**Reply by yydgis** [reply]
yes, for all the solutions in your folders, the final errors are so big, the NRCAN PPP solution is mm-cm accuracy. Do you know what is going on?
